hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/RepairOverwriteHoodiePropsProcedure.scala

```diff
@@ -57,7 +67,7 @@ class RepairOverwriteHoodiePropsProcedure extends BaseProcedure with ProcedureBu
     val metaClient = HoodieTableMetaClient.builder.setConf(jsc.hadoopConfiguration()).setBasePath(tablePath).build
 
     var newProps = new Properties
-    newProps.load(new FileInputStream(overwriteFilePath))
+    loadNewProps(overwriteFilePath, newProps)
```

Comment:
So this line does not work as expected?

Reply:
@danny0405 hi.When we use it, we usually specify the path where the new properties are located, such as hdfs://xxx/xxxx . The original implementation encountered an exception where the file could not be found. https://issues.apache.org/jira/browse/HUDI-7260

Reply:
I didn't see the exception handling with the new code either.

Reply:
hi, introduction in Fileinputstream code: `What files are available depends on the host environment.`.
When using `repair_overwrite_hoodie_props`, it is necessary to specify the location of the properties. Obviously, using FileInputStream will not be able to read data.

Reply:
hi, introduction in Fileinputstream code: What files are available depends on the host environment..
When using repair_overwrite_hoodie_props, it is necessary to specify the location of the properties. Obviously, using FileInputStream will not be able to read data.

Reply:
What we get for an empty file with this line:
`val fis = fs.open(new Path(filePath))`

Reply:
@danny0405  If the file content is empty, return empty properties. If the file does not exist, throw an exception stating that the file does not exist

Reply:
Thanks, then what about this line: `newProps.load(new FileInputStream(overwriteFilePath))`

Reply:
> https://issues.apache.org/jira/browse/HUDI-7260

`File does not exist: hdfs://xxx/user/xxx/tmp.db/spark_create_hudi_cow_partition_timestamp1/hoodie.properties`
https://issues.apache.org/jira/browse/HUDI-7260

Reply:
So you are saying both the new and old code throw the similiar exception, then why this change?

Reply:
hi, maybe I didn't express myself clearly before. Let me summarize, could you please help me take a look. The previous implementation was to read user specified files through `FileInputStream`. At this time, there will be a file does not exist exception. So I made modifications to this area and used `FSDataInputStream [fs.open(new Path(filePath))]` to read the data from the specified file.

Reply:
But you are saying `FSDataInputStream [fs.open(new Path(filePath))]` also throws exception for non-existing files?